_data/people/savvy379.yml

```
@@ -5,4 +5,16 @@ active: green
 institution: Princeton University
 website: 
 photo: /assets/images/team/Savannah-Thais.jpg
-presentations:
\ No newline at end of file
+presentations:
+  - title: "Introduction and Plans"
+    date: 09/12/19
+    url: https://indico.cern.ch/event/840472/contributions/3563440/attachments/1907910/3151492/iris_intro.pdf
+    meeting: IRIS-HEP Institute Retreat
+    location: Fermilab, Chicago IL
+    focus-area: ia
+  - title: "Semantic Segmentation for CMS Pixel Clustering"
+    date: 10/16/19
+    url: https://indico.cern.ch/event/844493/contributions/3601262/attachments/1928404/3193177/uslua_2019.pdf
+    meeting: US LHC Users' Association Meeting
```

```suggestion
    meeting: "US LHC Users' Association Meeting"
```

This scares me, let's add quotes, just to be sure.